project/AppDependencies.scala

```diff
@@ -20,11 +20,12 @@ object AppDependencies {
     "uk.gov.hmrc"       %% s"play-conditional-form-mapping$playSuffix" % "3.2.0",
     "org.typelevel"     %% "cats-core"                     % "2.12.0",
     "uk.gov.hmrc.mongo" %% s"hmrc-mongo$playSuffix"         % hmrcMongoVersion,
-    "uk.gov.hmrc"       %% s"crypto-json$playSuffix"        % "8.1.0"
+    "uk.gov.hmrc"       %% s"crypto-json$playSuffix"        % "8.2.0"
   )
 
   val test: Seq[ModuleID] = Seq(
-    "org.mockito"             %% "mockito-scala-scalatest"      % mockitoScalatestVersion,
+//    "org.mockito"             %% "mockito-scala-scalatest"      % mockitoScalatestVersion,
+    "org.scalatestplus"       %% "mockito-4-11"                 % "3.2.18.0",
```

Comment:
Can remove i think https://github.com/hmrc/bootstrap-play/blob/main/project/LibDependencies.scala#L27 or should be able to